Makefile

```diff
@@ -94,6 +100,12 @@ helm-install: helm-version check-openai-key kind-load-docker-images
 		--set app.image.tag=$(APP_IMAGE_TAG) \
 		--set openai.apiKey=$(OPENAI_API_KEY)
 
+.PHONY: helm-uninstall
+helm-uninstall:
+	helm uninstall kagent --namespace kagent
+	helm uninstall kagent-crds --namespace kagent
+
 .PHONY: helm-publish
 helm-publish: helm-version
+	helm push kagent-crds-$(VERSION).tgz oci://ghcr.io/kagent-dev/kagent/helm
```

Comment:
Note to self: This may fail the release since the OCI repo won't exist beforehand, I can solve that if this happens.

Reply:
Thanks @EItanya ,  please help if that happen.